packages/osd-opensearch/src/artifact.js

```diff
@@ -188,8 +188,8 @@ async function getArtifactSpecForSnapshotFromUrl(urlVersion, log) {
   const arch = process.arch === 'arm64' ? 'arm64' : 'x64';
   const extension = process.platform === 'win32' ? 'zip' : 'tar.gz';
 
-  if (platform !== 'linux' && platform !== 'windows') {
-    throw createCliError(`Snapshots are only available for Linux and Windows`);
+  if (platform !== 'linux' && platform !== 'windows' && platform !== 'darwin') {
```

Comment:
could we refactor for readability and maintainability 

```suggestion
const supportedPlatforms = ['linux', 'windows', 'darwin'];
if (!supportedPlatforms.includes(platform)) {

```

Reply:
Good suggestion! Modified it to be a const at the top.